setup.py

```diff
@@ -51,21 +51,25 @@ def run(self):
     license="Apache 2.0",
     description="Cloudstate Python Support Library",
     packages=packages,
+    package_data={
+        # If any package contains *.proto files, include them:
```

Comment:
I think it's clear what this code does so we can delete that comment

Reply:
removed